examples/index.json

```diff
@@ -264,14 +264,14 @@
     },
     {
         "title": "Network Interaction",
-        "column": 0,
+        "column": 1,
         "category": "Language Concepts",
         "samples": [
             {
                 "name": "Consuming Services: Client Objects",
                 "url": "consuming-services",
                 "verifyBuild": true,
-                "verifyOutput": false
+                "verifyOutput": true
```

Comment:
Let's not enable this. This is failing in PR builds for some reason. Need to check why.

Reply:
Fixed.